API/api-gestion.yml

```diff
@@ -522,19 +522,98 @@ paths:
                 examples:
                   error:
                     value: >- 
-                      {"error": "User not authenticated."}
-  /trabajadores/{id}:
+                      {"error": "User not authorized."}
+      
+      post:
+        summary: Inserta un nuevo trabajador
+        tags:
+        - Trabajadores
+        requestBody:
+            required: true
+            content:
+              multipart/form-data:
+                schema:
+                  type: object
+                  required:
+                    - name
+                    - lastname
+                    - telf
+                    - email
+                    - rol
+                    - dni
+                    - password
+                  properties:
+                    dni:
+                      type: string
+                    name:
+                      type: string
+                    lastname:
+                      type: string
+                    email:
+                      type: string
+                    telf:
```

Comment:
puede ser int

Reply:
El tema era guardar el +34622332234, y como un número al cual no vamos a hacer nada más que mostrarlo, decidimos dejarlo en string.

Pero si queréis podría ser un Long

Reply:
En ese caso si, string